apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/virusskanning/VirusScanService.kt

```diff
@@ -0,0 +1,26 @@
+package no.nav.etterlatte.brev.virusskanning
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+private const val MAKS_FILSTOERRELSE = 300 * 1024 * 1024
+
+private val log: Logger = LoggerFactory.getLogger(VirusScanService::class.java)
+
+class VirusScanService(private val clamAvClient: ClamAvClient) {
+    suspend fun filHarVirus(request: VirusScanRequest): Boolean {
+        log.info("Skanner fil for virus: ${request.tittel}")
+        if (filErForStor(request.fil)) {
+            log.info("Fil er for stor: ${request.tittel}")
+            return false
```

Comment:
Mulig jeg leser dette feil men dette betyr at hvis filen er for stor så dropper vi virussjekk men går fortsatt videre å lagrer pdf? 
Skal vi heller la være å tillate så store filer? 🤔

Reply:
@chrissearle kom meg i forkjøpet